package.json

```diff
@@ -6,7 +6,7 @@
   "proxy": "http://localhost:4200",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "concurrently \"webpack-dev-server --config  webpack.dev.js --host 0.0.0.0 --inline\" \"nodemon server.js\"",
+    "start": "concurrently \"webpack-dev-server --config  webpack.dev.js --host 0.0.0.0 --inline\" \"supervisor server.js\"",
```

Comment:
If we're dockering we might as well run two separate containers.

Reply:
that's a fair point